resources/javascripts/login.js

```diff
@@ -0,0 +1,56 @@
+var validation = false;
+
+callback = function (flag, id, value) {
+    validation = flag;
+    document.getElementById(id).innerHTML = value;
+}
+
+
+document.getElementById("submit-login").addEventListener("click", function () {
+    let ele = document.getElementById('login-form');
+    ele.childNodes.forEach(function (element) {
+        if (element.nodeName == "INPUT") {
+            if (document.getElementById(element.id).value == "") {
+                callback(false, 'error', "Fields can not be blank.");
+            }
+        }
+    });
+    if (validation) {
+        window.location.href = window.location.origin + "/";
+    }
+});
+
+
+function validate(e) {

```

Comment:
have a separate utility function which can be imported in both login and register files